packages/contentstack-config/src/config/remove-config.ts

```diff
@@ -0,0 +1,8 @@
+import { cliux, configHandler } from '@contentstack/cli-utilities';
+
+export async function removeConfig(apiKey) {
+  await configHandler.set(`baseBranch.${apiKey}`, 'main');
```

Comment:
Please delete the key for removal config handler has delete method to delete

Reply:
have added the changes